bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +23,285 @@ class CoffeeLintBear:
+        :param enforce_newline_at_EOF:
+            Checks if the file ends with a single newline.
+        :param use_spaces:
+            Forbids tabs in indentation and applies two spaces for this purpose.
+        :param tab_width:
+            Length of the tab for indentation.
+        :param newlines_after_classes:
+            Checks if there is number equal ``number_of_lines_after_classes`` of
+            newlines after classes definition.
+        :param number_of_newlines_after_classes:
+            Determines the number of newlines that separate the class definition
+            and the rest of the code.
+        :param no_backticks:
+            Prevents some JavaScript elements like ``eval`` to affect
+            CoffeeScript.
+        :param no_implicit_braces:
+            Prohibits implicit braces when declaring object literals.
+
+            Example: If ``no_implicit_braces = True`` then
+            ```
+            1:2, 3:4
+            ```
+            is prohibited, whereas
+            ```
+            {1:2, 3:4}
+            ```
+            is accepted.
+        :param no_implicit_parentheses:
+            Prohibits implicit parentheses.
+        :param no_interpolation_in_single_quotes:
+            Prohibits string interpolation in a single quoted string.
+
+            Example: If ``no_interpolation_in_single_quotes = True`` then
+            ```
+            f = '#{bar}'
+            ```
+            is prohibited, whereas
+            ```
+            f = "#{bar}"
+            ```
+            is correct.
+        :param no_stand_alone_at:
+            Prohibits the use of stand alone  ``@``.
+
+            Example: If ``no_stand_alone_at = True``
+            ```
+            @ notok
+            not(@).ok
+            @::
+            ```
+            are prohibited, whereas
+            ```
+            @alright
+            @(fn)
+            @ok()
+            @[ok]
+            @ok()
+            ```
+            are accepted.
+        :param disable_throwing_strings:
+            Disables throwing string literals or interpolation. It is forbidden
+            by default.
+
+            Example: If ``disable_throwing_strings = True``
+            ```
+            throw 'my error'
+            throw "#{1234}"
+            ```
+            will not be permitted.
+        :param allow_trailing_semicolons:
+            Prohibits trailing semicolons when ``False`` since they are
+            not useful. The semicolon is meaningful as there's another
+            instruction on the same line.
+
+            Example: If ``allow_trailing_semicolon = False``
+            ```
+            x = '1234'; console.log(x)
+            ```
+            here the semicolon is meaningful.
+            ```
+            alert('end of line');
+            ```
+            the semicolon is redundant.
+        :param allow_trailing_whitespaces:
+            Checks whether they are trailing whitespace in the code or not.
```

Comment:
Checks whether ***to allow*** ~~they are~~ trailing whitespaces***s*** in the code or not.